hardhat.vyper-config.ts

```diff
@@ -0,0 +1,6 @@
+// this file is used to configure the vyper compiler in hardhat config files
+export const vyperConfig = {
+  compilers: [
+    { version: "0.3.7" }
```

Comment:
just noting that this is NOT a bugged Vyper version with malfunctioning reentrancy locks.
(bugged versions are 0.2.15, 0.2.16, 0.3.0 per https://twitter.com/vyperlang/status/1685692973051498497)

Reply:
Appreciate you flagging ser 🙏